packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.styles.ts

```diff
@@ -307,9 +307,9 @@ export const getStyles = (props: IDetailsHeaderStyleProps): IDetailsHeaderStyles
       {
         display: 'none',
         position: 'absolute',
-        top: -18,
-        left: -7.5,
-        fontSize: fonts.mediumPlus.fontSize,
+        top: -28,
+        left: -6.5,
```

Comment:
Make sure this point values behave the same in the entire range of supported browsers

Reply:
Yes, browsers round in different ways. Tho I think only in more precise cases. Worth validating for sanity :).